index.bs

```diff
@@ -3103,17 +3103,26 @@ Each {{WebRequestEvent}} has:
   To <dfn>process beforeRequest events</dfn> given a [=request=] |request|,
   run the following steps:
 
-  1. Let |handlers| be the result of calling [=lookup registered WebRequest
-      handler configs=] given "beforeRequest", and |request|.
+  1. Let |handlers| and |blocking| be the results of calling
```

Comment:
could you clarify where |blocking| is coming from? is this coming from the same call?

Reply:
The algorithm this is calling has two return values.